library/eclipse-temurin

```diff
@@ -338,89 +338,88 @@ Constraints: nanoserver-1809, windowsservercore-1809
 #------------------------------v21 images---------------------------------
 Tags: 21.0.1_12-jdk-alpine, 21-jdk-alpine, 21-alpine
```

Comment:
@gdams shouldn't this be updated to `21.0.2_13` as well?  (or is that one of the builds that's still pending?)

Reply:
that build is still pending, I think we're safe to merge without)

Reply:
Alpine for JDK21 is also good to go now. @plebcity can you get this updated please and include the JDK17 updates

Reply:
I grabbed the output of the dockerhub_doc_config_update.sh file and replaced the eclipse-temurin file with the output. So it should all be updated now.

Reply:
@plebcity run it again, we've published JDK8u now